### DISCUSSION TITLE
Create offical sandboxes with Keystone examples projects and refer them in Docs

### DISCUSSION THREAD

**Discussion by MurzNN** [keystonejs/keystone#7222] [Feature requests]
Will be good to provide ready to use sandboxes with Keystone Example projects, for example using [codesandbox.io](https://codesandbox.io/) service or any other (stackblitz.com, glitch.com, etc).
They will be very useful to live test examples, described in Keystone Docs, without needs to creating own project, prove some theories, use them as Demo of Keystone interface, and much more...

And we can even link to that sandboxes in the Keystone Docs, to allow see live results of some API call or GraphQL query in user's browser instantly.

**Comment by MurzNN** [comment]
For example, we can quickly create a sandbox from `examples/blog`, regarding to this manual https://codesandbox.io/docs/importing, via generating a link like this:
https://githubbox.com/keystonejs/keystone/tree/main/examples/blog
But maybe it'll be more convenient to export examples to separate Git repos.

StackBlitz also provide very powerful API https://developer.stackblitz.com/docs/platform/importing-projects/#stackblitz-api for live creating of sanboxes.

**Comment by MurzNN** [comment]
Here https://quasar.dev/vue-components/table is a good example of documentation with links to live sandboxes for each example, where users can instantly try to play with this example in live sandboxes environment.

**Comment by MurzNN** [comment]
I've made a PR https://github.com/keystonejs/keystone/pull/7463 with adding support for codesandbox.io sandboxes, example: https://codesandbox.io/s/github/MurzNN/keystone/tree/codesandbox-configs/examples/blog

**Reply by dcousens** [reply]
Thanks @MurzNN!  This is really appreciated by the Keystone team and undoubtedly the Keystone community.  Thanks for the effort you have put in for this.
